packages/compiler/server/serverlib.ts

```diff
@@ -606,6 +613,31 @@ export function createServer(host: ServerHost): Server {
     }
   }
 
+  async function getDocument(params: DefinitionParams): Promise<string> {
+    const sym = await compile(params.textDocument, (program, document, file) => {
```

Comment:
That compile should also go in `getHoverSymbol` and this funciton should take in a `Type`

Reply:
it should really be `getTypeDetails(type: Type): string`